model_utils/tests/models.py

```diff
@@ -4,25 +4,24 @@
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 
+from model_utils import Choices
+from model_utils.fields import SplitField, MonitorField, StatusField
+from model_utils.managers import QueryManager, InheritanceManager
 from model_utils.models import (
     SoftDeletableModel,
     StatusModel,
     TimeFramedModel,
     TimeStampedModel,
 )
-from model_utils.tracker import FieldTracker, ModelTracker
-from model_utils.managers import QueryManager, InheritanceManager
-from model_utils.fields import SplitField, MonitorField, StatusField
 from model_utils.tests.fields import MutableField
-from model_utils import Choices
-
+from model_utils.tests.managers import CustomSoftDeleteManager
+from model_utils.tracker import FieldTracker, ModelTracker
 
 
 class InheritanceManagerTestRelated(models.Model):
     pass
 
 
-
```

Comment:
Years ago I used to prefer 3 blank lines between classes, but I've been following PEP8 for a long time now, just never did a full update of model-utils. PR welcome, or I'm fine letting it happen bit by bit. I don't really have much time for maintenance of model-utils these days, would be happy if someone wanted to take it over :-)

And yes, certainly we could add flake8 or pep8 or something to the Travis config, PR welcome for that too!